content/en/docs/components/pipelines/run-a-pipeline.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "Run a Pipeline"
+description = "Execute a pipeline on the KFP backend"
+weight = 6
++++
+
+The KFP SDK offers three ways to run a pipeline.
+
+## 1. Run from the KFP Dashboard
```

Comment:
If KFP dashboard the official term to reference that UI? I often refer to it as KFP Pipeline templates page. Not sure if we have a formal name for it.

Reply:
I think we refer to the FE in several ways. I think that that the v2 docs is a good opportunity to unify around one term we prefer. What do you prefer?

Reply:
I don't have a strong preference. Dashboard sounds good to me. Like to see others' opinion. 
/cc @zijianjoy

Reply:
Merging, but we can change this easily if other opinions emerge